superduperdb/backends/sqlalchemy/metadata.py

```diff
@@ -102,10 +40,89 @@ def __init__(
     ):
         self.name = name
         self.conn = conn
-        Base.metadata.create_all(self.conn)
+        self.dialect = conn.dialect.name
+        self.metadata = MetaData()
+        self._init_tables()
 
         self._lock = threading.Lock()
 
+    def _init_tables(self):
```

Comment:
Using this way of creating tables has better compatibility than using class objects.

Reply:
A driver of clickhouse uses the class object method to build a table and does not support updates.

Reply:
Please can you explain this `MetaData` in more detail?

Reply:
MetaData is used for storing and managing the definitions of tables, and this is a common practice for this way to create tables

https://docs.sqlalchemy.org/en/20/tutorial/metadata.html

Reply:
Done, move the metadata into `_init_tables`